### ISSUE TITLE
Installation unclear

### ISSUE THREAD

**Issue by romgrk** [yamatsum/nonicons#4]
Hi,

It's unclear how to install the icons. Could you add more details?

**Comment by p00f** [comment]
the ttf is bundles in `dist`

**Comment by romgrk** [comment]
Yep. But it's unusual. When we install any of the nerdfonts, they come as the font we want plus the patched icons, eg `DejaVu Sans Nerd Font`. This is a single `nonicon.ttf`, so should I just install it and be done? Because below is the result I get when I do that. Or should I change my terminal font to nonicon.ttf? But then I don't get the font I want... So yeah this is unusual. Are you using the same codepoints as the nerdfonts? Are you using other codepoints? How does it work to scale the icons for my text font?

![Screenshot from 2021-03-15 10-01-17](https://user-images.githubusercontent.com/1423607/111165618-a02e7880-8575-11eb-8163-8fd9bd3f3e2a.png)

Also [nvim-nnonicons](https://github.com/yamatsum/nvim-nonicons) doesn't mention that it needs to be setup with `lua require'nvim-nonicons'` to work.

**Comment by p00f** [comment]
I added nonicon as a fallback font in my terminal

**Comment by romgrk** [comment]
That would be a nice information to have on the readme.

Any plans to make patched fonts? I don't think any of my terminals have fallback font support.

**Comment by yamatsum** [comment]
Added to readme!
When it comes to preparing patch fonts, there is the issue of which font to prepare.
I don't have time to patch all the famous fonts. waiting for PR!
I would like to consider patch fonts if I have time.

**Comment by romgrk** [comment]
Ok sure, you can probably re-use [nerdfont's script](https://github.com/ryanoasis/nerd-fonts/blob/master/font-patcher). I'll wait for patched fonts then.

**Comment by meain** [comment]
Just had a related question. I have installed and set this in iTerm but I am guessing the codepoints(might not be the exact term) is not similar to what is being used in nerdfonts. For example, the rust icon is not available at the same glyph location that it would be in nerd fonts. Was that something you have explicitly decided not to do? If not, it would be better to keep them more or less same as that way a lot of tools like [lsd](https://github.com/Peltoche/lsd) that use nerd font can easily work with both.
